infra/gcp/main.tf

```diff
@@ -441,6 +441,12 @@ module "ci_gsa_secret" {
   name = "ci"
 }
 
+module "query_gsa_secret" {
+  source = "./gsa_k8s_secret"
+  name = "query"
+  iam_roles = ["storage.admin"]
+}
```

Comment:
What's this for? How is `gsa_k8s_secret` populated?

Reply:
I'll move into a separate PR.